src/types/theme.d.ts

```diff
@@ -27,5 +31,6 @@ declare module '@emotion/react' {
       red: string;
       backgroundGradient: string;
     };
+    themeFonts: typeof themeFonts;
```

Comment:
```suggestion
   fonts: typeof themeFonts;
```

다음과 같이 위의 colors랑 통일하는 게 어떨까요?

Reply:
좋습니다 수정하겠습니다!

Reply:
저기 아래에 commit suggestion을 누르면 바로 할 수 있땁니다!